WordPress/Classes/ViewRelated/Cells/TitleBadgeDisclosureCell.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  SiteTagCell.swift
+//  WordPress
+//
+//  Created by Cesar Tardaguila Moro on 2017-12-04.
+//  Copyright © 2017 WordPress. All rights reserved.
+//
```

Comment:
We like to omit these headers. :)

Reply:
Ouch! This one hurts, I've removed them form other files, I missed this one. Sorry!